tests/other/metadce/hello_libcxx_fastcomp_O2.sent

```diff
@@ -1,6 +1,5 @@
 ___gxx_personality_v0
 ___map_file
-___setErrNo
```

Comment:
what mechanism caused this change? Does being a JS-only symbol change linking (and not just warnings)?

Reply:
It means the symbols is no available to native code anymore so it makes that that it would not be "sent" to that native code anymore.   My guess is to why only the fastcomp is effected is that the way to detect "sent" functions is more accurate for llvm backend and only includes symbols directly used by the native code (which his symbol never is, obviously).

Reply:
Oh right, yes, thanks!